PROVESFlightControllerReference/test/int/radio_test.py

```
@@ -0,0 +1,118 @@
+lora = "ReferenceDeployment.lora"
+
+LORA_ERROR_EVENTS = ("SendFailed", "ConfigurationFailed", "AllocationFailed")
+
+# Continuous-wave burst duration (seconds) for the CW regression test. Kept short so
+# the command stays within the GDS command-completion timeout while still exercising
+# the asynchronous TxTimeout teardown path.
+CW_SECONDS = 5
+
+
+@pytest.fixture(autouse=True)
+def setup_test(fprime_test_api: IntegrationTestAPI, start_gds):
```

Ruff (0.15.20)

[warning] Unused function argument: `start_gds`

(ARG001)